importAllData

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+set -u
+./museum/importObjects -r -p -m
```

Comment:
Can't these two lines both be condensed to `./museum/importObjects -rpmf`?  Mainly, is there a reason the `-f` call is separate?  Looking at the script it looks like this will run the import twice?